dm/util.dm

```diff
@@ -1,7 +1,169 @@
+#define NUM_E 2.71828183
+#define NUM_SQRT2 1.41421356
 
-#define ceil(x) (-round(-(x)))
-#define floor(x) round(x)
-#define clamp(x, low, high) max((low),min((high),(x)))
+#define PI						3.1415
+#define SPEED_OF_LIGHT			3e8		//not exact but hey!
+#define SPEED_OF_LIGHT_SQ		9e+16
+#define INFINITY				1e31	//closer then enough
```

Comment:
You can actually define this as `1.#INF` instead.

Reply:
inf has some odd properties, people can use it directly if they want but most uses at least in ss13 code is just "very large number"